code/modules/reagents/reagent_dispenser.dm

```diff
@@ -126,10 +126,10 @@
 
 /obj/structure/reagent_dispensers/fueltank/attack_hand()
 	if (rig)
-		usr.visible_message("[usr] begins to detach [rig] from \the [src].", "You begin to detach [rig] from \the [src]")
-		if(do_after(usr, 20))
-			usr.visible_message("<span class='notice'>[usr] detaches [rig] from \the [src].</span>", "<span class='notice'>You detach [rig] from \the [src]</span>")
-			rig.loc = get_turf(usr)
+		user.visible_message("[user] begins to detach [rig] from \the [src].", "You begin to detach [rig] from \the [src]")

```

Comment:
fuck this bug again. `user` is not defined in this scope, despite the compiler not complaining. Add `mob/user` to the proc arguments.

Reply:
fixed